department-of-veterans-affairs/vets-website
# Pension 68561 mvp step 2 military info remove fields

## Summary
On the Pension Benefits application(21P-527EZ),
On the Step 2 Military History Chapter

Remove fields

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Proceed to Step 2 by filling out the required fields on Step 1
4. On the Step 2 - Service Periods page enter a service period that includes September 11, 2001
5. Verify that you are not asked 'Did you serve in a combat zone after 9/11/2001?' on the next page
6. Verify that you are not asked about the Reserve and National Guard on the next page
7.  Verify that the next page starts with 'Did you serve under another name?'
8. Validate data entry for 'Yes' and 'No'
9. Verify that the next page is titled 'POW status'
10. Verify that there are no questions about severance pay

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68561)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

### Desktop - Did you serve under another name?
| Before | After | Before(Yes) | After(Yes) | Error | Valid |
| ------ | ------ | ------ | ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_military_general](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1de6d85b-aeba-4769-9865-13bd53d47350)|![localhost_3001_pension_application_527EZ_military_general (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/19525edf-a6df-4c0f-83db-0020fd9da741)|![localhost_3001_pension_application_527EZ_military_general (before)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/eac8656d-3a27-40b9-a6e7-671d0af858c6)|![localhost_3001_pension_application_527EZ_military_general (after)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ed896e17-6af3-4870-9359-7f7345b968c0)|![localhost_3001_pension_application_527EZ_military_general (error)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/0c76201d-aacb-49bd-9dd8-e8506e385bcf)|![localhost_3001_pension_application_527EZ_military_general (valid)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/93239bd6-d4d7-4873-995e-862e6936c793)



### Desktop - POW status
| Before | After  | After(Yes) |
| ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_military_general (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/d38846cf-392c-4e05-a65b-03b9b1b9d5b7)|![localhost_3001_pension_application_527EZ_military_pow-severance](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ad22d844-9195-43a8-8c0f-2da6742903e9)|![localhost_3001_pension_application_527EZ_military_pow](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/f1cb4998-8dfb-4150-a873-e0085d7286ce)



## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
The following questions will be removed from Step 2 Military Info section:

- Did you serve in a combat zone after 9/11/2001?
- Reserve and National Guard. Are you currently on federal active duty in the National Guard? (Step 2.2)
- Have you received any type of severance or separation pay?
- Pay type
- Amount

### Quality Assurance & Testing

### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
